tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -68,6 +77,27 @@ def _construct_reconfiguraiton_wedge_status(self):
         reconf_msg.additional_data = bytes()
         return reconf_msg
 
+    def _construct_reconfiguraiton_prune_request(self, latest_pruneble_blocks):
+        prune_cmd = cmf_msgs.PruneRequest()
+        prune_cmd.sender = 1000
+        prune_cmd.latest_prunable_block = latest_pruneble_blocks
+        reconf_msg = cmf_msgs.ReconfigurationRequest()
+        reconf_msg.command = prune_cmd
+        reconf_msg.signature = bytes()
+        reconf_msg.additional_data = bytes()
+        return reconf_msg
+
+    def _construct_reconfiguraiton_prune_status_request(self):
```

Comment:
Nit: change `reconfiguraiton` to `reconfiguration`.

Reply:
done